### PR TITLE
Fix sending steps to cross device client

### PR DESCRIPTION
**Pull request by hjheath**
Bad bug, not sure when I introduced it!

### CODE REVIEW

**Comment by rfreitas** [comment]
Travis automatic deployment: https://227-pr-onfido-sdk-ui-onfido.surge.sh

File: src/index.ejs [hunk 1]
```diff
@@ -48,12 +48,6 @@
         if (options.async === "false") {
           getToken(createSDK)
         }
-        else if (options.token) {
-          createSDK(options.token)
-        }
-        else if (options.mobileFlow) {
-          window.onfidoOut = Onfido.init({mobileFlow: true})
```
Comment (stefaniacardenas) [inline src/index.ejs]: So where do you set `mobileFlow` to `true` now?

Comment (hjheath) [reply src/index.ejs]: This is set in the hosted_web_sdk repo: https://bitbucket.org/onfido/hosted_web_sdk/src/4547923844801f855109d14892f0a73ac1efb334/production.html?at=master&fileviewer=file-view-default

This index.ejs is only used for development. The code I removed here is legacy from the proof of concept

Comment (stefaniacardenas) [reply src/index.ejs]: Oh yes, forgot about that.

**Pull request merged**